server_manager/README.md

```diff
@@ -41,7 +41,7 @@ npm run action server_manager/electron/build ${PLATFORM} -- --buildMode=[debug,r
 
 Where `${PLATFORM}` is one of `linux`, `macos`, `windows`.
 
-The per-platform standalone apps will be at `output/build/server_manager/electron/static/dist`.
+The per-platform standalone apps will be at `output/build/server_manager/electron/${PLATFORM}/dist`.
```

Comment:
Perhaps `output/build/server_manager/${PLATFORM}`? I don't think `electron` does much here.
We may want to differentiate the different artifacts though, like the static www, vs unpacked electron project vs the final binary...

Reply:
It depends. Every output that is platform-dependent should be in a platform directory.

www can be outside if it's platform-agnostic, but I think it may be not.

Reply:
It only makes sense to have the platform-agnostic www if we are reusing outputs and doing incremental builds. It is fine to put it in the platform directory, it's probably safer. Notice that the electron builder will copy it into the unpacked, and eventually to dist, so there will be multiple copies, which is expected.